docs/en/latest/config.json

```diff
@@ -12,6 +12,18 @@
       {
         "type": "doc",
         "id": "the-internal-of-apisix-java-plugin-runner"
+      },
+      {
+        "type": "doc",
+        "id": "installation-guide"
+      },
+      {
+        "type": "doc",
+        "id": "deployment-guide"
+      },
+      {
+        "type": "doc",
```

Comment:
I'm not sure it's a good idea to add an empty document to the documentation

Reply:
`config.json` is used to display sidebar on the official website, see: https://apisix.apache.org/docs/java-plugin-runner/development/